lib/jekyll-gist/gist_tag.rb

```diff
@@ -42,6 +47,31 @@ def gist_script_tag(gist_id, filename = nil)
         end
       end
 
+      def gist_noscript_tag(gist_id, filename = nil)
+        code = fetch_raw_code(gist_id, filename)
+        if !code.nil?
+          "<noscript><pre>#{CGI.escapeHTML(code)}</pre></noscript>"
+        else
+          Jekyll.logger.warn "Warning:", "The <noscript> tag for your gist #{gist_id} could not"
+          Jekyll.logger.warn "", "be generated. This will affect users who do not have"
+          Jekyll.logger.warn "", "JavaScript available or enabled in their browsers."
+        end
+      end
+
+      def fetch_raw_code(gist_id, filename = nil)
+        if filename.empty?
+          uri = "https://gist.githubusercontent.com/#{gist_id}/raw"
+        else
+          uri = "https://gist.githubusercontent.com/#{gist_id}/raw/#{filename}"
+        end
```

Comment:
I think it does look cleaner. 
I guess the same could be done for the `gist_script_tag` method:
``` ruby
def gist_script_tag(gist_id, filename = nil)
  url = "https://gist.github.com/#{gist_id}.js"
  url = "#{url}?file=#{filename}" unless filename.empty?
  "<script src=\"#{url}\"> </script>"
end
```